react-components/src/apps/admin/AllStats/AppStatsComponents/NavBarTabs.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Imported in AllStats.js. This function imports the files for all the tabs that are present under TabPanels
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { AppBar, Tabs, Tab, Typography, Box } from "@material-ui/core";
+import JobsTab from "./JobsTab";
+import AppsTab from "./AppsTab";
+import UsersTab from "./UsersTab";
+import ids from "./AllStatsIDs";
+import { getMessage, withI18N, build } from "@cyverse-de/ui-lib";
+import myMessagesFile from "./messages.js";
+
+function TabPanel(props) {
+    const { children, value, baseId, index, ...other } = props;
+
+    return (
+        <Typography
+            component="div"
+            role="tabpanel"
+            hidden={value !== index}
+            id={`nav-tabpanel-${index}`}
+            aria-labelledby={`${baseId}.${index}`}
+            {...other}
+        >
+            {value === index && <Box p={3}>{children}</Box>}
+        </Typography>
+    );
+}
+
+TabPanel.propTypes = {
+    children: PropTypes.node,
+    index: PropTypes.any.isRequired,
+    value: PropTypes.any.isRequired,
+};
+
+function a11yProps(baseId, index) {
+    return {
+        id: build(baseId, index),
+        "aria-controls": `${baseId}-nav-tabpanel-${index}`,
```

Comment:
Starting a new thread because it was getting harder to find the old one 😄 
This is still not quite right. In their example, there are only 2 IDs that they're dealing with: 'simple-**tabpanel**-${index}' and 'simple-**tab**-${index}'. And what you need to do is get them to match between your **tab** and **tabpanel** code.  At this stage, you have 3 different IDs, which is incorrect.  

Here's what I'm talking about from MUI's example to show what's supposed to match:
<img width="460" alt="image" src="https://user-images.githubusercontent.com/8909156/74075774-7ea5a980-49d1-11ea-8503-e0eb7c683d05.png">

Since you're not using an actual index, I would rename your variables because maybe that's also causing confusion. 
Ultimately, you just need something like `build(baseId, tabName)` and either `build(baseId, panelName)` **or** `build(baseId, tabName, "panel")` if you want to do something similar to their example. Please let me know if this is still not making sense.